batch_correction/batch_correct_combat.Rmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "Batch Correction"
+author: "ALSF CCDL - Candace Savonen"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+*Purpose*: This notebook illustrates how to assess batch effects and how to use
+ComBat to correct for them. 
+
+This method is based on:
+Johnson, WE, Rabinovic, A, and Li, C (2007). Adjusting batch effects in microarray 
+expression data using Empirical Bayes methods. Biostatistics 8(1):118-127.
+
+## 1) Set up libraries
+Install `sva` if you haven't done so before.
+
+```{r eval=FALSE}
+source("https://bioconductor.org/biocLite.R")
+biocLite("sva", suppressUpdates = TRUE)
+```
+
+```{r Upload ComBat functions}
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+# Load ggplot2 for graphing
+library(ggplot2)
+```
+
+## 2) Import and set up data
+Data downloaded from refine.bio include a metadata tab separated values ("tsv")
+file and a data tsv file. 
+These files were obtained from [refine.bio](https://www.refine.bio/) by entering
+the search term "HEK 293" and then aggregating by `species`.
+
+```{r Import refine.bio data}
+# Put name of data file
+data.filename <- file.path("..", "data", "HOMO_SAPIENS.tsv")
+
+# Unzip the file and direct it to the data folder
+unzip(paste0(data.filename, ".zip"), exdir = file.path("..", "data"))
+
+# Read in data tsv file
+df <- readr::read_tsv(data.filename) %>% 
```

Comment:
Can we set `progress = FALSE` here? Since it's such a big file, showing progress in the notebook is a lot of lines.